packages/react-examples/src/react-accordion/AccordionItem/AccordionItem.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {
+  AccordionItem,
+  AccordionHeader,
+  AccordionPanel,
+  Accordion,
+  AccordionHeaderSize,
+  AccordionHeaderExpandIconPosition,
+} from '@fluentui/react-accordion';
+import { ChevronRightIcon, ChevronDownIcon } from '@fluentui/react-icons-mdl2';
+
+const ExpandIcon = ({ open, ...rest }: { open: boolean }) => {
+  return open ? <ChevronDownIcon {...rest} /> : <ChevronRightIcon {...rest} />;
+};
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface AccordionItemExampleProps {
```

Comment:
Should we simply reuse there props from `Accordion`?

```
interface AccordionItemExampleProps extends AccordionProps {
 label: string;
}
```

Reply:
They're not 1:1, there're props from `Accordion` + `AccordionItem` + `AccordionHeader`. I could do a union of types but I'm afraid there might be some conflicts.